backup/main.go

```diff
@@ -53,9 +41,7 @@ func Main() {
 
 	app := cli.NewApp()
 	app.Version = VERSION
-	app.Commands = []cli.Command{
-		backupRestoreCmd,
-	}
+	app.Commands = []cli.Command{}
```

Comment:
We can remove the unnecessary codes for cmdline as well.

Reply:
Do you mean to say that removing the entire main function from this file is better?

Reply:
Yeah.